package.json

```diff
@@ -30,7 +30,13 @@
       "@semantic-release/commit-analyzer",
       "@semantic-release/release-notes-generator",
       "@semantic-release/npm",
-      "@semantic-release/github"
+      "@semantic-release/github",
+      [
+        "@semantic-release/exec",
+        {
+          "prepareCmd": "echo ${nextRelease.version} > .version"
```

Comment:
By outputting the .version file we can check it before we do our second npm publish to ensure we only publish IF semantic release released a version.